.github/workflows/closed_issue_check.yml

```diff
@@ -0,0 +1,51 @@
+# Contains job to check that all the todos are resolved for the closed issue.
+
+name: Closed Issue Check
+
+on:
+  issues:
+    types: [closed, reopened]
+
+permissions:
+  issues: write
+
+jobs:
+  script_check:
+    name: Script Check
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Bazel
+        uses: abhinavsingh/setup-bazel@v3
+        with:
+          version: 4.0.0
+
+      - name: Closed Issue Check
+        id: closedIssueCheck
+        if: ${{ github.event.action == 'closed' }}
+        run: |
+          bazel run //scripts:closed_issue_check -- $(pwd) ${{ github.event.issue.number }} ${{ github.sha }}
+
+      - name: Reopen Issue
+        if: failure() && steps.closedIssueCheck.outcome == 'failure'
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: |
+          gh issue reopen ${{ github.event.issue.number }}
+
+      - name: Check for duplicate comment
+        id: duplicateCommentCheck
+        if: failure() && steps.closedIssueCheck.outcome == 'failure'
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: |
+          gh issue view ${{ github.event.issue.number }} --json comments --jq '.comments[-1].body' > $(pwd)/latest_comment.txt
+          bazel run //scripts:comment_check -- $(pwd) latest_comment.txt todo_list.txt
```

Comment:
Is todo_list.txt the correct name for this file? It seems more like it contains the contents for an issue comment.

Reply:
Corrected.